lib/index.js

```diff
@@ -1,93 +1,20 @@
 'use strict'
 
-const Ipinfo = require('ipinfo')
-const RequestIp = require('request-ip')
+const IpLocator = require('./ip-locator')
 
 /**
- * Set the initial state for `location` decorator
- *
- * @returns {undefined}
+ * @async
```

Comment:
Please remove the comments here as well (there's a comment in the ip-locator.js)